model/project_ref.go

```diff
@@ -1482,26 +1479,20 @@ func GetTasksWithOptions(projectName string, taskName string, opts GetProjectTas
 	return res, nil
 }
 
-func FindFirstProjectRef() (*ProjectRef, error) {
-	projectRefSlice := []ProjectRef{}
-	pipeline := projectRefPipelineForValueIsBool(ProjectRefPrivateKey, RepoRefPrivateKey, false)
-	pipeline = append(pipeline, bson.M{"$sort": bson.M{ProjectRefDisplayNameKey: -1}}, bson.M{"$limit": 1})
-	err := db.Aggregate(
-		ProjectRefCollection,
-		pipeline,
-		&projectRefSlice,
-	)
-
+// FindDefaultProjectRef returns an unrestricted project to use as a default for contexts.
+func FindDefaultProjectRef() (*ProjectRef, error) {
```

Comment:
nit: might be better to call it something more explicit like `FindAnyUnrestrictedProjectRef` or something like that since the results aren't ordered/can change, meaning the project it returns can be a little arbitrary.

Reply:
Makes sense!